core/arch/arm/include/mm/tee_mmu_defs.h

```diff
@@ -32,7 +32,7 @@
 #define TEE_MMU_TTBCR_N_VALUE		7
 
 /* Number of sections in ttbr0 when user mapping activated */
-#define TEE_MMU_UL1_NUM_ENTRIES         ((1 << TEE_MMU_TTBCR_N_VALUE) / 4)
+#define TEE_MMU_UL1_NUM_ENTRIES         (1 << (12 - TEE_MMU_TTBCR_N_VALUE))
```

Comment:
@jenswi-linaro   I think we can just modity the TEE_MMU_TTBCR_N_VALUE to change the user TA address space, is that right?